tests/unit/test_spds_agent_additional.py

```
@@ -1,11 +1,11 @@
+import builtins
+import json
 import re
```

Flake8 (7.2.0)

[error] 're' imported but unused

(F401)